napalm/eos/utils/cli_syntax.py

```diff
@@ -0,0 +1,357 @@
+        "show port-channel dense": "show port-channel summary",
+        "show port-channel load-balance": "show port-channel traffic",
+        "show port-security mac-address": "show port-security address",
+        "show ptp local-clock": "show ptp clock",
+        "show ptp masters": "show ptp parent",
+        "show ptp local-clock time properties": "show ptp time-property",
+        "show redundancy status": "show redundancy states",
+        "show users roles": "show role",
+        "show snmp v2-mib chassis": "show snmp chassis",
+        "show snmp v2-mib contact": "show snmp contact",
+        "show snmp notification host": "show snmp host",
+        "show snmp v2-mib location": "show snmp location",
+        "show snmp local-interface": "show snmp source-interface",
+        "show snmp notification": "show snmp trap",
+        "show spanning-tree instance": "show spanning-tree bridge",
+        "show users accounts": "show user-account",
+        "show vlan brief count": "show vlan summary",
+        "snmp trap link-change": "snmp trap link-status",
+        "snmp-server local-interface": "snmp-server source-interface",
+        "spanning-tree transmit active": "spanning-tree bridge assurance",
+        "spanning-tree guard loop default": "spanning-tree loopguard default",
+        "spanning-tree edge-port bpdufilter default": "spanning-tree portfast bpdufilter default",
+        "spanning-tree edge-port bpduguard default": "spanning-tree portfast bpduguard default",
+        "spanning-tree bpdu tx hold-count": "spanning-tree transmit hold-count",
+        "spanning-tree vlan-id": "spanning-tree vlan",
+        "counters per-entry": "statistics per-entry",
+        "switchport backup-link": "switchport backup interface",
+        "switchport port-security mac-address maximum": "switchport port-security maximum",
+        "switchport vlan translation": "switchport vlan mapping",
+        # disabled next command because of ambiguity
+        # 'timers': 'timers basic',
+        "timers lsa rx min interval": "timers lsa arrival",
+        "timers lsa tx delay initial": "timers throttle lsa all",
+        "timers spf delay initial": "timers throttle spf",
+        "username ssh-key": "username sshkey",
+        "vlan internal order": "vlan internal allocation policy",
+        "vrf instance": "vrf definition",
+        # disabled next command because of ambiguity
+        # 'vrf': 'vrf forwarding',
+        "vrrp peer authentication": "vrrp authentication",
+        "vrrp timers delay reload": "vrrp delay reload",
+        "vrrp session description": "vrrp description",
+        "vrrp ipv4": "vrrp ip",
+        "vrrp ipv4 secondary": "vrrp ip secondary",
+        "vrrp priority-level": "vrrp priority",
+        "vrrp disabled": "vrrp shutdown",
+        "vrrp advertisement interval": "vrrp timers advertise",
+        "vrrp tracked-object": "vrrp track",
+    },
+    # CLI syntax version after EOS 4.23.0
+    2: {
```

Comment:
Are all these the mirror of the above? In order to have less stuff to maintain, what if we just flip what's under `1`? Like ``CLI_SYNTAX[2] = {val, key for key, val in CLI_SYNTAX[1].items()}``?

Anyhow, my understanding is that we should only convert, going forward, from the old command to the new command format - and not vice-versa?

Reply:
I had the same conversation when I first reviewed this, and this was Alexey's feedback:


> Nice idea, however, we have to remove some commands from 2->1 translation dict because of ambiguity. For example, 'description': 'ip msdp description'. It's hard to determine is it related to msdp description or interface description without tracking config mode. And tracking config mode is a bit tough task.

Regarding conversion, I had the following thought:

> Would be nice if the converter supported 2 -> 1 as well, for backwards compatibility with new NAPALM code (someone writes a new getter that uses new commands, will it run on old?)

Reply:
Sounds good